app/assets/stylesheets/views/_html-publication.scss

```diff
@@ -31,85 +55,81 @@
     }
   }
 
-  .publication-header {
-    @include white-links;
-    @include core-19;
-    color: $white;
-
-    background: $light-blue;
-    padding: $gutter $gutter $gutter / 3;
-    margin-bottom: $gutter;
-
-    @include media(tablet) {
-      margin-bottom: $gutter * 2;
-    }
+  .offset-one-quarter {
+    margin-left: 25%;
 
-    &.direction-rtl {
-      direction: rtl;
-      text-align: start;
+    .direction-rtl & {
+      margin-left: 0;
+      margin-right: 25%;
     }
-  }
-
-  .headings {
-    margin-bottom: $gutter;
-  }
 
-  .in-page-navigation {
-    margin-bottom: $gutter;
+    @include media(mobile) {
+      margin: 0;
 
-    @include media(tablet) {
-      width: 70%;
+      .direction-rtl & {
+        margin: 0;
+      }
     }
+  }
 
-    h2 {
-      font-weight: bold;
+  .contents-list-wrapper {
+    .direction-rtl & {
+      direction: rtl;
+      text-align: start;
     }
 
-    ol {
-      padding-left: $gutter;
-
-      .direction-rtl & {
-        padding-right: $gutter;
-        padding-left: 0;
-      }
+    .contents-list {
```

Comment:
I think so, yes. I looked for alternatives but couldn't find any suitable option. It could have been done as a variant on the existing dash-list but that would have meant having a version like '.dash-list.without-dashes', which seemed confusing.